app/static/ringing_room.js

```diff
@@ -802,11 +806,13 @@ $(document).ready(function () {
                        disabled: lock_controls}"
                        @click="set_tower_size(size)"
                        >
+                    <!--
                     <input type="radio"
                            class="autoblur"
                            name="size"
                            :value="size"
                            />
+                    -->
```

Comment:
@lelandpaul I'm not sure what this code does - but I do know that including it generates a 2nd `c_size_change` signal (thus making the peal-speed adjustment be applied twice, reaching the incorrect value).  Also removing it seems to have no effect....

Reply:
OK I've pushed a change to fix this (although it's more of a band-aid than an actual fix).  If you do end up fixing it, then let me know and I'll revert d16c667.